src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -611,6 +611,13 @@ export class Repl extends ViewPane implements IHistoryNavigationWidget {
 			this.replInput.focus();
 			return Promise.resolve();
 		}));
+		actions.push(new Action('debug.replPaste', localize('paste', 'Paste'), undefined, true, async () => {
+			const clipboardText = await this.clipboardService.readText();
```

Comment:
Make sure this action is before the `Collapse All` action.
So it should be Copy, Copy All, Paste, Collapse All